.circleci/config.yml

```diff
@@ -1,164 +1,113 @@
-          arangodb-version: "arangodb:3.10.10"
-          arangodb-config: "single-3.10.conf"
-          cluster: false
-          enterprise: false
-  python-3.8-enterprise-cluster-3.10:
-    when: << pipeline.parameters.3-10 >>
-    jobs:
-      - run-tests:
-          name: python-3.8-enterprise-cluster-3.10
-          python-version: "3.8.2"
-          arangodb-version: "enterprise:3.10.10"
-          arangodb-config: "cluster-3.10.conf"
-          cluster: true
-          enterprise: true
-  python-3.10-community-single-3.11:
-    when: << pipeline.parameters.3-11 >>
-    jobs:
-      - run-tests:
-          name: python-3.10-community-single-3.11
-          python-version: "3.10.6"
-          arangodb-version: "arangodb:3.11.4"
-          arangodb-config: "single.conf"
-          cluster: false
-          enterprise: false
-  python-3.10-community-cluster-3.11:
-    when: << pipeline.parameters.3-11 >>
-    jobs:
-      - run-tests:
-          name: python-3.10-community-cluster-3.11
-          python-version: "3.10.6"
-          arangodb-version: "arangodb:3.11.4"
-          arangodb-config: "cluster.conf"
-          cluster: true
-          enterprise: false
-  python-3.10-enterprise-single-3.11:
-    when: << pipeline.parameters.3-11 >>
-    jobs:
-      - run-tests:
-          name: python-3.10-enterprise-single-3.11
-          python-version: "3.10.6"
-          arangodb-version: "enterprise:3.11.4"
-          arangodb-config: "single.conf"
-          cluster: false
-          enterprise: true
-  python-3.10-enterprise-cluster-3.11:
-    when: << pipeline.parameters.3-11 >>
-    jobs:
-      - run-tests:
-          name: python-3.10-enterprise-cluster-3.11
-          python-version: "3.10.6"
-          arangodb-version: "enterprise:3.11.4"
-          arangodb-config: "cluster.conf"
-          cluster: true
-          enterprise: true
-  python-latest-enterprise-single-preview:
-    when: << pipeline.parameters.preview >>
-    jobs:
-      - run-tests:
-          name: python-latest-enterprise-single-preview
-          python-version: "latest"
-          arangodb-version: "enterprise-preview:latest"
-          arangodb-config: "single.conf"
-          cluster: false
-          enterprise: true
-  python-latest-enterprise-cluster-preview:
-    when: << pipeline.parameters.preview >>
-    jobs:
-      - run-tests:
-          name: python-latest-enterprise-cluster-preview
-          python-version: "latest"
-          arangodb-version: "enterprise-preview:latest"
-          arangodb-config: "cluster.conf"
-          cluster: true
-          enterprise: true
+      # - run:
+      #     name: Upload to Coveralls
+      #     command: |
+      #       if [ "<< parameters.python_version >>" = "3.11" && "<< parameters.arangodb_config >>" = "single" && "<< parameters.arangodb_license >>" = "community" && "<< parameters.arangodb_version >>" = "latest"  ]; then
+      #         coveralls/upload
+      #       fi
```

Comment:
Uploading coverage won't be possible until we request access from the Organization Admin:

![image](https://github.com/ArangoDB-Community/python-arango/assets/43019056/d7c1058d-e28f-405d-8fe0-d88c56f831ea)